home/service/details.py

```diff
@@ -194,3 +196,69 @@ def _get_context(self):
             ),
             "PlatformUrns": PlatformUrns,
         }
+
+
+class PublicationCollectionDetailsService(GenericService):
+    def __init__(self, urn: str):
+        self.urn = urn
+        self.client = self._get_catalogue_client()
+
+        self.publication_collection_metadata = self.client.get_publication_collection_details(self.urn)
+
+        if not self.publication_collection_metadata:
+            raise ObjectDoesNotExist(urn)
+
+        self.entities_in_container = self.publication_collection_metadata.relationships[
+            RelationshipType.CHILD
+        ]
+        self.context = self._get_context()
+        self.template = "details_publication_collection.html"
+
+    def _get_context(self):
+        context = {
+            "entity": self.publication_collection_metadata,
+            "entity_type": EntityTypes.PUBLICATION_COLLECTION.value,
```

Comment:
On main there is another change that's gone in to add `PlatformUrns` to the context, can you add that in to these ones for consistency?